TFUtil.py

```diff
@@ -150,7 +152,7 @@ def __init__(self, name,
     if vocab is not None:
       from GeneratingDataset import Vocabulary
       if isinstance(vocab, str):
-        vocab = Vocabulary(vocab)
+        vocab = Vocabulary(vocab, unknown_label=unknown_label)
```

Comment:
As said, not needed. If you want to specify this, you can just use the case which you find just in the next code line.

Reply:
I can't use  Vocabulary(**vocab) because my requirement is that the vocab is specified just as a vocab filename in the config file, and is loaded exactly in this line here.

Reply:
I don't understand. You can specify the vocab filename + the unknown label in the config. Where is the problem?